src/jekyll/_plugins/custom_tags/ytvideo.rb

```diff
@@ -54,9 +54,14 @@ def render(context)
         iframeSrcUrl += '&amp;' + argument
       end
 
-      out = '<div class="embed">'
-      out += '<iframe src="' + iframeSrcUrl + '" frameborder="0" allowfullscreen=""></iframe>'
-      out += "</div>"
+      #out = '<div class="embed">'
```

Comment:
Unless we really want to keep these commented out lines, let's yank them completely.